server/request_processor.go

```diff
@@ -199,12 +193,16 @@ func (r *RpcRequest) sendTxToRelay() {
 	// only allow large transactions to certain addresses - default max tx size is 128KB
 	// https://github.com/ethereum/go-ethereum/blob/master/core/tx_pool.go#L53
 	if r.tx.Size() > 131072 {
-		if _, found := allowedLargeTxTargets[txTo.Hex()]; !found {
-			r.logger.Error("[sendTxToRelay] large tx not allowed to target", "tx", txHash, "target", txTo)
+		if r.tx.To() == nil {
```

Comment:
@metachris For larger tx with nil To address, we still set it as sentToRelay (True) in redis even though this tx is not sent to redis. I assume we are ok with that.

Reply:
Yes, that's fine just as before, it's just used for spam protection - to prevent resending the same tx over and over. If this tx would be sent again it would run into the same error, so that's fine.